src/plugin/styled-components/App.js

```diff
@@ -1,8 +1,14 @@
 import React from "react";
 import AppOriginal from "AppOriginal";
+import GlobalStyles from "../../styles/globalStyles";
 
 function App() {
-  return <AppOriginal />;
+  return (
+    <>
+      <GlobalStyles />
```

Comment:
@zdavis I've added the same GlobalStyles here that we have in Manifold Marketing, so there's less repetition of class names and styles in the SASS files.